docs/FAQ.rst

```diff
@@ -0,0 +1,286 @@
+is not unsupported.
+
+Q: Where can I find the list of operations that DALI supports?
+##############################################################
+A: You can find a comprehensive list of operators available `here <../supported_ops.html>`_.
+
+Q: Can I send a request to the Triton server with a batch of samples of different shapes (like files with different lengths)?
+#############################################################################################################################
+A: Even though DALI supports such data, Triton doesn't accept non-uniform batches. You can approach
+it in two ways. The first is to pad the samples to make a uniform batch. The second is to split
+the request so you send together only samples with the same shape - in some cases, they can be
+later internally merged in the Triton server thanks to the dynamic batching feature.
+
+Q: I have heard about the new data processing framework XYZ, how is DALI better than it?
+########################################################################################
+A: DALI is a library that aims to GPU accelerate certain workloads we see that suffer the most
+due to being CPU bottleneck. There are many cases not covered by DALI, or where DALI
+can be suboptimal, and these are the places where other solutions could shine.
+
+What is worth remembering, there is a lot of advertised optimizations in other libraries that
+come with the cost of lower accuracy in the training or inference process - DALI has proved
+itself in MLPerf benchmarks and `NVIDIA Deep Learning Examples <https://github.com/NVIDIA/DeepLearningExamples>`_
+where not only speed but also accuracy matters. So the user is sure that DALI doesn't cut corners.
+
+Q: Is DALI compatible with other GPUs?
+######################################
+A: When it comes to the question if DALI supports non-NVIDIA GPUs, the answer is no.
+DALI GPU implementations are written in CUDA. However there are open source community efforts
+that are enabling running CUDA-based applications on other GPU architectures, but DALI
+doesn't officially support it.
+
+Q: When to use DALI and when RAPIDS?
+####################################
+A: RAPIDS is better suited for general-purpose machine learning and data analytics.
+DALI is a specialized tool for Deep Learning workflows, and it's aimed to accelerate dense data
+processing and to overlap the preprocessing with the network forward/backward passes.
+
+Q: Is Triton + DALI still significantly better than preprocessing on CPU, when minimum latency i.e. batch-size = 1 is desired?
+##############################################################################################################################
+A: That can depend on the particular case and what other tool we are comparing to.
+One of DALI's advantages is the fact that it can optimize batch processing,
+but even for batch-size = 1, it might be worth using DALI because of the GPU implementation.
```

Comment:
```suggestion
Q: Is Triton + DALI still significantly better than preprocessing on CPU, when minimum latency i.e. batch_size=1 is desired?
##############################################################################################################################
A: That can depend on the particular case and what other tool we are comparing to.
One of DALI's advantages is the fact that it can optimize batch processing,
but even for batch_size=1, it might be worth using DALI because of the GPU implementation.
```

Reply:
Fixed